### DISCUSSION TITLE
Test Request works differently open source version and hosted

### DISCUSSION THREAD

**Discussion by paulmeller** [scalar/scalar#921] [Q&A]
Is this expected behavior? I'm using the same Swagger doc and getting 2 different response outputs:

<img width="1448" alt="Screenshot 2024-02-01 at 1 48 44 PM" src="https://github.com/scalar/scalar/assets/14568/1973acaa-08ac-4b11-9d82-9489f80702ac">
<img width="1451" alt="Screenshot 2024-02-01 at 1 51 45 PM" src="https://github.com/scalar/scalar/assets/14568/847b5662-ace8-40fb-af57-2c42146d59b3">

**Comment by marclave** [comment]
ah! this is most likely because we haven't updated on proxy on the hosted version, I will kick off an update on that today.

I will ping here once it's live, thanks for reporting this @paulmeller

**Comment by paulmeller** [comment]
For clarity, the behavior of the hosted version is correct.

**Reply by hanspagel** [reply]
Oops, there were recent changes to the proxy, we probably broke it. I’ll look into it.
